### ISSUE TITLE
Coordinate destination Player training: representation pretraining -> RL policy learning

### ISSUE THREAD

**Issue by rhythmatician** [rhythmatician/FractalSync#130]
Part of #77.

Coordinates with #104 (PlayerPolicy), #108/#109 (observation/audio contracts), #107 (Controls v2), #110 (reset distribution), #120 (manifold Physics), #95 (Julia presentation), #111 (spectator/evaluation), #102/#103 (offline teachers), and later reward research such as #128.

- **Status:** open
- **Type:** wayfinder:task / parent coordination
- **Purpose:** own the common causal training substrate and the explicit handoff between representation learning and behavioral policy learning

## Child phases

- [ ] #131 — **Phase 1: Pretrain the causal Player representation from training-only supervision**
- [ ] #132 — **Phase 2: Train the Controls-v2 Player with gradient-based RL**

The split is deliberate:

```text
#131 Phase 1
learn to perceive
teacher / representation losses
        |
        v
versioned causal trunk checkpoint
        |
        +-------------------------+
                                  v
#132 Phase 2                 #132 baseline
learn to act                 RL from random init
behavioral reward                  |
        |                           |
        +------------ compare -----+
```

## Key dependency rule

**#132 is not globally blocked by #131.**

Phase-2 RL from random initialization should be brought up independently so we have a real baseline. Only the pretrained-initialization arm depends on a compatible #131 checkpoint.

That yields the primary transfer experiment:

```text
RL from random initialization
vs
RL from #131 pretrained trunk
```

under matched observation, reward, reset, compute, and evaluation conditions where practical.

## Why the split exists

The two phases solve different optimization problems and have different failure modes.

### #131 asks

> **What should the Player's causal hidden state understand about what is happening in the music?**

Learning signal:

```text
direct teacher / self-supervised targets
-> differentiable representation loss
-> ordinary backpropagation
```

No Controls target or behavioral reward is required.

### #132 asks

> **Given what the Player understands and its current world state, what should it do?**

Learning signal:

```text
Controls v2
-> Rust environment trajectory
-> behavioral consequences / reward
-> policy-gradient / actor-critic update
```

No supervised correct action is assumed.

Keeping these separate lets a bad Player be diagnosed as a representation problem, a reward/exploration/credit-assignment problem, or both rather than hiding all causes inside one end-to-end objective.

## Shared architectural invariants

Both children consume the same destination contracts rather than inventing trainer-local alternatives:

- #109 `AudioActivation/1`;
- #108 versioned `PlayerObservation`;
- #104 causal PlayerPolicy / hidden-state semantics;
- #107 Controls-v2 schema for the product policy;
- #120 Rust-authoritative Mandelbrot manifold Physics;
- #95/#107 Rust-owned Julia presentation integration;
- #110 reset-state semantics where world rollouts are involved;
- canonical sample-clock timing;
- ADR 0001/#119 Rust-first authority.

Do not preserve the legacy orbit-control trainer as a second product architecture.

## Common causality invariant

At tick `t`, model input/state may depend only on information causally available through `t`.

```text
h_t = f_theta(O_t, h_{t-1})
```

Offline/noncausal teachers may use future or whole-song information only on the target/loss side in #131 or optional explicitly ablated auxiliary losses in #132.

Changing future audio may change a teacher target. It may not change an already-constructed causal observation or hidden state.

## Canonical chronology

The shared model chronology uses canonical analysis/sample time, not render time.

For closed-loop #132 rollouts:

```text
O_t = observation(audio_{<=t}, world_t, julia_view_t)

(h_t, controls_t) = PlayerPolicy(O_t, h_{t-1})

(world_{t+1}, julia_view_{t+1})
    = Rust destination step(world_t, julia_view_t, controls_t, dt)
```

Chunking/minibatching may truncate gradients. It must not silently reset forward hidden/world state.

## Important differentiability correction

Gradient-based learning does **not** imply differentiating through the entire FractalSync environment.

- #131 uses ordinary backpropagation through the model and auxiliary heads.
- #132's baseline uses gradient-based RL while treating Rust Physics/presentation as a black-box environment.
- A differentiable Physics forward mirror is **not** a prerequisite for PPO/policy-gradient/actor-critic training.
- Any later environment-gradient experiment must satisfy ADR 0001/#119 and justify why Rust-forward + explicit backward/surrogate treatment is insufficient.

## Shared checkpoint / schema handoff

#131 must produce a checkpoint that #132 can consume without semantic adaptation.

The handoff should stamp enough metadata to fail closed on incompatible versions, including at least:

```text
AudioActivation version
PlayerObservation version
PlayerPolicy/trunk architecture version
hidden-state contract version
training-phase/checkpoint version
```

#132 additionally stamps Controls, Physics, reset, reward/evaluator, and Julia-presentation contract versions as required for reproducible rollouts.

## Attribution / ablation contract

The baseline experiment matrix must preserve attribution:

```text
A. RL from random initialization
B. #131 pretraining -> RL
C. #131 pretraining -> RL + retained auxiliary losses   [optional ablation]
```

Optional variants such as an initially frozen pretrained trunk may be tested later.

Do not make retained auxiliary losses in Phase 2 implicit. Their value must be measured separately.

## Common evaluation boundary

Auxiliary prediction accuracy does not establish that #131 helped the visualizer.

The decisive transfer evidence is downstream #132 behavior under matched conditions, including where practical:

- RL sample efficiency;
- stability across seeds;
- held-out song/performance behavior;
- transition preparation/timing;
- scale/realm/energy competence;
- action/chatter quality;
- eyes-on coherence through #111.

Frozen #132 evaluation rollouts must be reproducible and replayable in #111 against the exact checkpoint/audio/reset/actions/state used for metrics.

## Legacy demolition

Across both children, destination training must eliminate dependence on legacy assumptions such as:

- orbit-target product outputs;
- target-`c` servo training;
- direct music-energy/transient Physics coupling;
- legacy observation layouts;
- positional model-output semantics;
- trainer-owned duplicate Rust state-transition math;
- future-aware runtime lookahead;
- the assumption that perception and control must be learned in one undifferentiated optimization pass;
- the assumption that gradient-based policy learning requires differentiable Physics.

Compatibility adapters may keep old experiments inspectable temporarily, but destination training must not depend on them.

## Parent acceptance

This parent resolves when:

- [ ] #131 is complete or explicitly closed with a negative result that still yields a usable no-pretraining baseline decision.
- [ ] #132 is complete with a working gradient-based RL destination Player baseline.
- [ ] #132 can run from random initialization without #131.
- [ ] If #131 produces a retained checkpoint, #132 consumes it without observation/state schema adaptation.
- [ ] Random-init vs pretrained-init receives a matched downstream comparison.
- [ ] Any retained Phase-2 auxiliary losses receive an explicit ablation rather than becoming implicit architecture.
- [ ] Causality, hidden-state chronology, and canonical-time semantics agree across both phases.
- [ ] Rust remains authoritative for deterministic runtime/environment semantics.
- [ ] Frozen held-out #132 evaluation is replayable through #111 with exact provenance.
- [ ] Legacy orbit-control training semantics are no longer required by either destination phase.

## Non-goals

- Do not duplicate the detailed #131 or #132 implementation plans here.
- Do not choose final AudioActivation, PlayerObservation, PlayerPolicy, Controls, or Physics contracts here.
- Do not implement SongFormer/affect experiments here; #102/#103 supply Phase-1 targets.
- Do not implement speculative entrainment reward here; #128 remains evidence-gated.
- Do not require neuroevolution/QD for the initial two-phase baseline; it remains a possible later optimization layer.
